src/apps/Hosting/Files.js

```diff
@@ -89,7 +89,7 @@ const HostingFilesView = React.createClass({
         <InnerToolbar title="Website Hosting">
           <Show if={items.length && !isLoading}>
             <RaisedButton
-              label={isDefaultHosting ? 'Published' : 'Publish'}
+              label={isDefaultHosting ? 'Default' : 'Set as default'}
```

Comment:
This button is replaced with `Go to site` button in this pr: https://github.com/Syncano/syncano-dashboard/pull/9 so either way it will disappear, so should I change it, or leave it?